dotnet/Hec.Dss/DssWriter.cs

```diff
@@ -465,5 +465,10 @@ public void Write(PairedData pd)
       DSS.ZpdStore(ref ifltab, ref pds, storageFlag);
     }
 
+    public int SetProgramName(string programName)
```

Comment:
Does the SetProgramName need to be on the writer?   Should this be a higher level static method such as how Zset is high level in DSS?